src/extras/CoolSHT1x.cpp

```diff
@@ -0,0 +1,251 @@
+/**
```

Comment:
Is that library anywhere on Github ?
If so, did you need to patch it ?
What does justify the need to copy-paste it here ?
I won't review anything more until you explain me those things.

Reply:
OK, so as far as I can see, this library is [GPL v3](https://github.com/practicalarduino/SHT1x/blob/master/DISTRIBUTION)

There is absolutely NO WAY we are copy-pasting this code to **this** repository where the licence headers state we distribute the code here under the **MIT** license !

cc @simon-lacool